### PR TITLE
New CONTRIBUTING.md | Local dev/tophatting instructions

### PR DESCRIPTION
**Pull request by taimoorgit**
These are useful instructions that explain how to tophat changes locally. It also gives a decent overview of how the program works.

**REVIEWERS- click here to view the rendered docs:**

1. https://github.com/OpsLevel/kubectl-opslevel/blob/ta/contributing-md/docs/Local-Development.md
2. https://github.com/OpsLevel/kubectl-opslevel/blob/ta/contributing-md/CONTRIBUTING.md
3. https://github.com/OpsLevel/kubectl-opslevel/blob/ta/contributing-md/Readme.md

### CODE REVIEW

File: docs/Local-Development.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+**Get an OpsLevel API token for a dev environment.** Do not use your prod environment since this involves mutating lots of services in OpsLevel.
+
+Install these:
+
+1. [minikube](https://minikube.sigs.k8s.io/docs/start/)  - this is a suggestion, you can also use **your own dev cluster if you have one.**
+1. [task](https://taskfile.dev/) - this will install dependencies for you.
+1. [OpsLevel CLI](https://github.com/OpsLevel/cli) - makes things way easier.
+
+```
+brew install minikube
```
Comment (rocktavious) [inline docs/Local-Development.md:19]: 😢 - do we really use minikube - https://kind.sigs.k8s.io/ is way better and less of a resource hog

File: docs/example-opslevel-k8s.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+version: 1.2.0
```
Comment (rocktavious) [inline docs/example-opslevel-k8s.yaml:1]: does this match the output of `kubectl opslevel config sample --simple` ?  If so could we recommend doing just that so we don't have to maintain this file if things about the config change?

**Review by rocktavious — APPROVED** [review]
Overall looks good just a few suggestions - but i'm good without them if i'm offbase for somereason.

File: docs/example-opslevel-k8s.yaml (hunk 2 above)
Comment (taimoorgit) [reply docs/example-opslevel-k8s.yaml:1]: No, this is an even simpler config I made that's about 10 lines shorter and just needs pods and not deploys so it's slightly easier to setup a test env for IMO. 

I will add a line telling them to use that command for another example of a config, but if we do change the config the output would change anyways and we'd have to re-write this file. 🙃

File: docs/Local-Development.md (hunk 1 above)
Comment (taimoorgit) [reply docs/Local-Development.md:19]: (: minikube is mentioned in a few k8s docs and I thought it was really easy to install even if it is a resource hog, I'll mention that is also an option along with it.

**Pull request merged**